groovy/postInit/chemistry/inorganic_chemistry/elements/s_block/group1/LithiumChain.groovy

```diff
@@ -257,9 +257,16 @@ CRYSTALLIZER.recipeBuilder()
 BR.recipeBuilder()
         .inputs(ore('dustLithiumCarbonate') * 6)
         .fluidInputs(fluid('nitric_acid') * 2000)
+        .fluidOutputs(fluid('lithium_nitrate_solution') * 1000)
```

Comment:
change this to 2kL LiNO3.H2O

Reply:
this would need additional 1000L of water input, is it sure to do it?

Reply:
change the formula of the LiNO3 solution to LiNO3.2H2O

Reply:
why do you add water to the reaction

Reply:
to get 1:1 ratio in solution

Reply:
![image](https://github.com/user-attachments/assets/f9f4a1d9-31db-421e-bfe8-e96f85c50fbc)

Reply:
don't do this 
revert back to 1 LiNO3 2 H2O

Reply:
done

Reply:
btw thats 2 LiNO3 1 H2O

Reply:
well looks like we're back to the starting point :trollface:
actually can we let it output LiNO3 dust directly
or if you don't want to bother with this ig this is pretty safe to go

Reply:
ah
2 LiNO3 1 H2O then